packages/fast-usdc/README.md

```diff
@@ -59,43 +59,50 @@ sequenceDiagram
 
 # Status Manager
 
-### Pending Advance State Diagram
+### State Diagram
 
-*Transactions are qualified by the OCW and EventFeed before arriving to the Advancer.*
+*Transactions are qualified by the OCW and TransactionFeed before being
+delivered to the Advancer.*
 
-```mermaid
-stateDiagram-v2
-  [*] --> Observed: observe()
-  [*] --> Advancing: advancing()
-
-  Advancing --> Advanced: advanceOutcome(...true)
-  Advancing --> AdvanceFailed: advanceOutcome(...false)
-
-  Observed --> [*]: dequeueStatus()
-  Advanced --> [*]: dequeueStatus()
-  AdvanceFailed --> [*]: dequeueStatus()
-
-  note right of [*]
-    After dequeueStatus():
-    Transaction is removed 
-    from pendingTxs store.
-    Settler will .disburse()
-    or .forward()
-  end note
-```
+The transactionFeed receives attestations from Oracles, records their
+evidence, and when enough oracles agree, (if no risks are identified)
+it publishes the results for the advancer to act on.
 
-### Complete state diagram (starting from Transaction Feed into Advancer)
+The Advancer subscribes (via `handleTransactionEvent`) to events published by
+the transactionFeed. When notified of an appropriate opportunity, it is
+responsible for advancing funds to fastUSDC payees.
+
+The Settler is responsible for monitoring (via `receiveUpcall`) deposits to the
+settlementAccount. It either `disburse`s funds to the Pool (if funds were
+`advance`d to the payee), or `forwards` funds to the payee (if pool funds
+were not `advance`d).
 
 ```mermaid
 stateDiagram-v2
-  Observed --> AdvanceSkipped : Risks identified
-  Observed --> Advancing : No risks, can advance
-  Observed --> Forwarding : No risks, Mint deposited before advance
-  Forwarding --> Forwarded
-  Advancing --> Advanced
-  Advanced --> Disbursed
-  AdvanceSkipped --> Forwarding : Mint deposited
-  AdvanceFailed --> Forwarding : Mint deposited
-  Advancing --> AdvanceFailed
-  Forwarding --> ForwardFailed
-```
+  [*] --> AdvanceSkipped : Risks identified
+  [*] --> Advancing : No risks, can advance
+  [*] --> Forwarding* : No risks, Mint deposited before advance
+
+  state Advancer  {
```

Comment:
this works visually but if someone knows this syntax, [composite states](https://mermaid.js.org/syntax/stateDiagram.html#composite-states), they might thing that these are composite states.

It's all states of the Transaction and not a composite. The boxes indicate what exo can set transition to that state.

I'm not sure that's useful but if you want to keep it please clarify in a comment on this sytnax.

Reply:
I thought it was clarifying to separate advancer actions from settler actions, but I can see how calling them composites would be confusing.

I also tried to find a way to use different colors for actions in Advancer and Settler, but I could only see how to color states, and that didn't carry the right meaning.